dataflux_pytorch/benchmark/checkpointing/simulated/llama2.py

```diff
@@ -0,0 +1,119 @@
+import os
+import socket
+import statistics
+import time
+
+import torch
+import torch.distributed as dist
+
+from demo.lightning.checkpoint.simulated.llama2 import (
```

Comment:
This doesn't need to be addressed here, but we should probably restructure our imports such that benchmark doesn't import directly from "demo" or vice versa. Imports probably need to be structured into a "components" or "building_blocks" location outside of both.

Reply:
Ack